src/main/java/uk/gov/hmcts/reform/sandl/snlrules/services/DroolsService.java

```diff
@@ -27,6 +27,7 @@
     public void init() {
         logger.info("Starting drools service, container and session for {}", droolsConfiguration.getRulesKSession());
         KieServices kieServices = KieServices.Factory.get();
+        //kieServices.newKieBaseConfiguration().setOption(EqualityBehaviorOption.EQUALITY);
```

Comment:
any good reason to leave it? or xml equality update is enough?

Reply:
this is intentional because it took about 3 hours to find how to do it, even though we use xml config to set the EQUALITY, in the feature when we have the persistence of rules implemented we may need to set it in the code. Therefore left it here,